docs/modules/ROOT/pages/connect-to-viridian.adoc

```diff
@@ -1,30 +1,49 @@
 == Connecting to {hazelcast-cloud} with Hazelcast CLC
-:description: To use the Hazelcast CLC with {hazelcast-cloud}, you can download a preconfigured client. You don’t need to do any configuration.
+:description: To use the Hazelcast CLC with {hazelcast-cloud}, you can import CLC configuration from a Viridian cluster. You don’t need to do any configuration.
 
 :page-product: cloud
 
 {description}
 
-The xref:configuration.adoc[configuration file] in a sample client provides the Hazelcast CLC with with the correct connection credentials.
+You can also create a cluster using CLC, which automatically makes the cluster configuration available. See Managing Viridian Clusters for more information.
```

Comment:
Feels like this should move to the end of the task, once users can start to do cluster operations. I've made another suggestion at the end.
```suggestion

```